package.json

```diff
@@ -108,6 +108,7 @@
     "@storybook/addon-actions": "^5.3.2",
     "@storybook/addon-console": "^1.2.1",
     "@storybook/addon-contexts": "^5.3.2",
+    "@storybook/addon-knobs": "^5.3.9",
```

Comment:
Match this version to the one we use for the other extensions - 5.3.2 - to avoid a bunch of yarn noise

Reply:
Still pretty noisy